src/test/java/com/twilio/http/LoggingTest.java

```diff
@@ -0,0 +1,37 @@
+package com.twilio.http;
+
+import com.twilio.Twilio;
+import com.twilio.rest.Domains;
+import com.twilio.rest.api.v2010.account.Message;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.Assert;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+public class LoggingTest {
+    @Before
+    public void setUp() throws Exception {
+        Twilio.init("AC123", "AUTH TOKEN");
+    }
+
+    @Test
+    public void testDebuggingLogging() {
```

Comment:
If behavior is different when no logger is configured (i.e. no output), can we add a test for that scenario?